src/components/paragraphs/rows/three-column.tsx

```diff
@@ -16,7 +16,7 @@ const ThreeColumn = ({items}: {items: ParagraphUnion[]}) => {
   }
 
   return (
-    <div className="centered grid gap-10 @12xl:grid-cols-3 md:gap-20" {...draftProps}>
+    <div className="centered grid gap-10 lg:grid-cols-3 lg:gap-20" {...draftProps}>
```

Comment:
Can we still use container queries? interior pages will have different breakpoints than full width pages.

Reply:
Ooo great catch! I'll update it to the container query version 🤔